data.sql

```diff
@@ -1,13 +1,118 @@
-/* Populate database with sample data. */
 -- data.sql
-USE vet_clinic;
 
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Agumon', '2020-02-03', 0, TRUE, 10.23),
-('Gabumon', '2018-11-15', 2, TRUE, 8),
-('Pikachu', '2021-01-07', 1, FALSE, 15.04),
-('Devimon', '2017-05-12', 5, TRUE, 11);
+-- Create the species table
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100) UNIQUE NOT NULL
+);
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+-- Create the owners table
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(100) NOT NULL,
+    age INTEGER
+);
+
+-- Create the animals table
+CREATE TABLE animals (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INTEGER NOT NULL,
+    neutered BOOLEAN NOT NULL,
+    weight_kg NUMERIC NOT NULL,
+    species_id INTEGER REFERENCES species(id),
+    owner_id INTEGER REFERENCES owners(id)
+);
+
+-- Create the vets table
+CREATE TABLE vets (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    age INTEGER NOT NULL,
+    date_of_graduation DATE NOT NULL
+);
+
+-- Create the specializations join table
+CREATE TABLE specializations (
+    vet_id INTEGER REFERENCES vets(id),
+    species_id INTEGER REFERENCES species(id),
+    PRIMARY KEY (vet_id, species_id)
+);
+
+-- Create the visits join table
+CREATE TABLE visits (
+    vet_id INTEGER REFERENCES vets(id),
+    animal_id INTEGER REFERENCES animals(id),
+    visit_date DATE NOT NULL,
+    PRIMARY KEY (vet_id, animal_id, visit_date)
+);
```

Comment:
- [ ] Please make sure that your SQL commands are distributed correctly in your files. As stated by the previous reviewer, any database-related SQL commands should only exist in the `schema.rb` file. 🙏